org.eclipse.scanning.api/src/org/eclipse/scanning/api/annotation/ui/FieldValue.java

```diff
@@ -212,7 +212,7 @@ private static Object get(Object model, Class<?> clazz, String name) throws Ille
 				}
 			}
 		}
-		
+
 		final String isser  = getIsserName(name).toLowerCase();
 		for (Method method : methods) {
 			if (method.getName().toLowerCase().equals(isser)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1157)